NitroxClient/GameLogic/ChatUI/PlayerChatManager.cs

```diff
@@ -0,0 +1,87 @@
+﻿using System.Collections;
+using NitroxClient.Communication.Abstract;
+using NitroxClient.MonoBehaviours.Gui.Chat;
+using NitroxClient.Unity.Helper;
+using NitroxModel.Packets;
+using UnityEngine;
+
+namespace NitroxClient.GameLogic.ChatUI
+{
+    public class PlayerChatManager
+    {
+        public PlayerChatManager(IMultiplayerSession multiplayerSession)
+        {
+            this.multiplayerSession = multiplayerSession;
+            Player.main.StartCoroutine(LoadChatLogAsset());
+        }
+
+        private PlayerChat playerChat;
+        public Transform PlayerChaTransform => playerChat.transform;
+        private readonly IMultiplayerSession multiplayerSession;
+
+        public void ShowChat() => Player.main.StartCoroutine(ShowChatAsync());
+        private IEnumerator ShowChatAsync()
+        {
+            while (PlayerChat.IsLoading)
+            {
+                yield return null;
+            }
```

Comment:
Instead of these while loops can we use: https://docs.unity3d.com/ScriptReference/WaitUntil.html (if so, please update all of the while loops in the PR)